src/hotspot/share/compiler/compilerOracle.hpp

```diff
@@ -58,6 +58,7 @@ class methodHandle;
   option(Break, "break", Bool) \
   option(BreakAtExecute, "BreakAtExecute", Bool) \
   option(BreakAtCompile, "BreakAtCompile", Bool) \
+  option(TriggerAtExecute, "TriggerAtExecute", Bool) \
```

Comment:
Should we use either Enter or Execute throughout?

Reply:
Maybe TriggerOnEnter

Reply:
EndTrainingOnEnter